unstructured/ingest/connector/box.py

```diff
@@ -27,36 +27,25 @@ class AccessTokenError(Exception):
 @dataclass
 class SimpleBoxConfig(SimpleFsspecConfig):
     @requires_dependencies(["boxfs"], extras="box")
-    def __post_init__(self):
+    def get_access_kwargs(self):
```

Comment:
This is really nice.

Reply:
Thanks! Yea, definitely needed to sit with this one a bit, but happy with where we landed.